projects/storefrontapp-e2e-cypress/cypress/helpers/update-profile.ts

```diff
@@ -23,12 +23,19 @@ export function updateProfile(user?: SampleUser) {
     );
     cy.get('[formcontrolname="lastName"]').should('have.value', user.lastName);
   }
+  cy.get('.myaccount-enhancedUI-editButton')
```

Comment:
maybe it's better to add new functions instead of modifying existing ones that for v1.

Reply:
two sides of the coin,  modify/reuse code make it easily if we wanna change sth in the future

Reply:
I don't get the point. Does V1 also have this element `myaccount-enhancedUI-editButton`? Won't it cause any failure in  V1‘s tests?